netlify/edge-functions/workspaces-card.tsx

```diff
@@ -0,0 +1,209 @@
+import React from "react";
+import { ImageResponse } from "og_edge";
+import type { Config } from "https://edge.netlify.com";
+// import { Pill } from "components/Pill.tsx";
+import {
+  ArrowTrendingUpIcon,
+  MinusSmallIcon,
+  ArrowTrendingDownIcon,
+} from "https://esm.sh/@heroicons/react@2.0.14/24/solid";
+
+const baseApiUrl = Deno.env.get("NEXT_PUBLIC_API_URL");
+console.log({ baseApiUrl });
+
+interface PillProps {
+  children: React.ReactNode;
+  color?: "slate" | "green" | "yellow" | "red" | "purple";
+}
+
+const Pill = ({ children, color = "slate" }: PillProps) => {
+  return (
+    <div
+      style={{
+        backgroundColor:
+          color === "green"
+            ? "hsl(121 47.5% 91.4% / 1)"
+            : color === "yellow"
+            ? "rgb(253 230 138 / 1)"
+            : color === "red"
+            ? "hsl(360 97.9% 94.8% / 1)"
+            : color === "purple"
+            ? "rgb(233 213 255 / 1)"
+            : "hsl(209 12.2% 93.2% / 1)",
+        color:
+          color === "green"
+            ? "hsl(133 50.0% 32.5% / 1)"
+            : color === "yellow"
+            ? "rgb(180 83 9 / 1)"
+            : color === "red"
+            ? "hsl(358 65.0% 48.7% / 1)"
+            : color === "purple"
+            ? "rgb(147 51 234 / 1)"
+            : "hsl(206 6.0% 43.5% / 1)",
+        borderRadius: "9999px",
+        padding: "0.375rem 0.5rem",
+      }}
+    >
+      {children}
+    </div>
+  );
+};
+
+function getLocalAsset(url: URL): Promise<ArrayBuffer> {
+  return fetch(url).then((res) => res.arrayBuffer());
+}
+
+const getPillChart = (total?: number): React.ReactNode => {
+  if (total === undefined) {
+    return "-";
+  }
+
+  if (total > 7) {
+    // icon={<ArrowTrendingUpIcon color="green" className="w-6 h-6 lg:w-4 lg:h-4" />}
+    return <Pill color="green">High</Pill>;
+  }
+
+  if (total >= 4 && total <= 7) {
+    //         icon={
+    //   <MinusSmallIcon
+    //     color="black"
+    //     style={{ background: "#fff", strokeColor: "#fff", width: 24, height: 24, border: "1px solid red" }}
+    //   />
+    // }
+
+    return <Pill color="yellow">Medium</Pill>;
+  }
+
+  // icon={<ArrowTrendingDownIcon color="red" className="w-6 h-6 lg:w-4 lg:h-4" />}
+  return <Pill color="red">Low</Pill>;
+};
+
+export default async function handler(req: Request) {
+  const { searchParams } = new URL(req.url);
+  const workspaceName = searchParams.get("wname");
+  const workspaceId = searchParams.get("wid");
+  const range = searchParams.get("range");
+  console.dir({ workspaceName, baseApiUrl });
+
+  const response = await fetch(new URL(`${baseApiUrl}/workspaces/${workspaceId}/stats?range=${range}`, baseApiUrl));
```

Comment:
getting the repo stats only in here. We already have the workspace info available to us from the page that the OG image is for.